### PR TITLE
Add display of math functions

### PR DESCRIPTION
**Pull request by juliusberger**
Add display of Math functions below the usual information.
Link to #92

### CODE REVIEW

**Comment by isolovev** [comment]
Hello. Ok.
Can you pull the master branch?

File: src/code/codeMaths.pug [hunk 1]
```diff
@@ -0,0 +1,5 @@
+mixin codeMaths()
+	pre
+		| #[span.u-color-second function #[span.u-color-brand.js-info-name]](x: number) {
```
Comment (isolovev) [inline src/code/codeMaths.pug:3]: Do you think we should add the type returned by the function?

Comment (juliusberger) [reply src/code/codeMaths.pug]: Ok, like I did now in 478a52b?

Comment (isolovev) [reply src/code/codeMaths.pug]: 👍

**Comment by isolovev** [comment]
You completed the task but did not change your PR status. Have you finished this PR yet?

**Comment by juliusberger** [comment]
I was not sure regarding the translations that we would need to set outdated for now. Or do we fill in the English wording into the other existing translations? I mean otherwise, the text is not shown, and it might be unclear that a translation might be necessary...

**Comment by isolovev** [comment]
Let's add the English translation for existing translations. I can add the translation for Russian.

**Comment by juliusberger** [comment]
Okay, done. When you finished russian translation, we can merge in my eyes.

**Comment by isolovev** [comment]
Thank you for your contribution to this project!

**Pull request merged**